src/pages/components/accordion/code.mdx

```diff
@@ -4,17 +4,13 @@ description: The accordion component delivers large amounts of content in a smal
 tabs: ['Code', 'Usage', 'Style', 'Accessibility']
 ---
 
-## Accordion
-
-<ComponentCode
-  name="Accordion"
-  component="accordion"
-  variation="accordion"
-  hasReactVersion="accordion--default"
-  hasVueVersion="accordion--default"
-  hasAngularVersion="?path=/story/accordion--basic"
-  codepen="WWrNvp"></ComponentCode>
-
-## Documentation
+<ComponentDemo
+  knobs={{ AccordionItem: ['open'] }}
+  code={`
+<Accordion>
+  <AccordionItem title="title 1">content 1</AccordionItem>
+</Accordion>
+`}
+/>
 
 <ComponentDocs component="accordion"></ComponentDocs>
```

Comment:
I think we want to remove all of the `<ComponentDocs />` components, they pull in vanilla docs.

Reply:
@alisonjoseph I opened https://github.com/carbon-design-system/carbon-website/pull/648 to take care of that along with replace the `ComponentCode` components.